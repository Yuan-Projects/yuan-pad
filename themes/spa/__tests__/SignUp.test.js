import renderer from 'react-test-renderer';
import SignUp from '../src/signUp';
import GuestUser from '../__mocks__/user-guest';
import LangEn from '../__mocks__/site-translate-en';

it('SignUp renders correctly', () => {
  let props1 = {
    user: GuestUser,
    lang: LangEn,
    onCurrentUserUpdated: () =>{}
  };
  const component1 = renderer.create(<SignUp
    {...props1}
  />);
  let tree1 = component1.toJSON();
  expect(tree1).toMatchSnapshot();
});