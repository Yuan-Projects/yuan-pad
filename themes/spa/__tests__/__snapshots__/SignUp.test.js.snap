// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders correctly 1`] = `
<div
  className="signUp"
>
  <a
    className="btn btn-default"
    onClick={[Function]}
    role="button"
  >
    Register
  </a>
</div>
`;
